### ISSUE TITLE
batch normalizationのmomentは0.5でいいと思う

### ISSUE THREAD

**Issue by yunicoder** [yunicoder/my-tiny-dnn#1]
表題の通り。

おそらくデータの平均と分散を求める際に移動平均を行っていて、その重みを`moment_`として定義されている。
https://github.com/yunicoder/my-tiny-dnn/blob/c0f576f5cb7b35893f62127cb7aec18f77a3bcc5/tiny_dnn/layers/batch_normalization_layer.h#L172-L178

デフォルト値は0.999になっており最初に処理したバッチでの値を重視しているが、0.5で平等に評価していいと思う。
実際にこの値を変えると精度も変わって、0.5の方が高い。
